Formula/postgis.rb

```diff
@@ -72,7 +72,13 @@ def install
       utils/test_joinestimation.pl
     ]
 
-    man1.install Dir["doc/**/*.1"]
+    man1.install dir("doc/**/*.1")
+  end
+
+  def dir(pattern)
```

Comment:
There will need to be a comment justifying the use of a special function for globing.

Reply:
Done.